### PR TITLE
navigation bar updated

### PR DESCRIPTION
**Pull request by ayushnanda21**
modified the nav panel please look into it

### CODE REVIEW

**Comment by parasg1999** [comment]
Would be great if you could attack some screenshots on desktop and mobile view, or host it on heroku

**Comment by ayushnanda21** [comment]
**_yes i have attached both desktop view and mobile views. Please let me know if any kind of modifications or chnages are required._**
Screenshot (583)
Screenshot (582)

**Comment by parasg1999** [comment]
Could you please align the items on mobile view to the left?

**Comment by ayushnanda21** [comment]
Sir, I have aligned items to left in mobile view please have a look and suggest if any modifications are required?
i have attached the snapshot. please let me know when should I make a pull request again, Thanks.
Screenshot (585)

**Comment by ayushnanda21** [comment]
Done

File: views/index.ejs [hunk 1]
```diff
@@ -18,26 +18,44 @@
...
+          <li class=" help nav-item active">
+            <a class="nav-link" href="#">
+              <i class="fa fa-phone-volume"></i>
+              Contact
+              <span class="badge badge-danger"></span>
+              </a>
+          </li>
+          <li class=" help nav-item active">
+            <a class="nav-link" href="/writing/postStory">
+              <i class="far fa-edit"></i>s
```
Comment (parasg1999) [inline views/index.ejs]: That `s` seems to be a typo

Comment (ayushnanda21) [reply views/index.ejs]: so sorry, I have updated it

**Comment by parasg1999** [comment]
As mentioned in #13, you wanted to link the empty tabs to some data.

**Comment by ayushnanda21** [comment]
That are just about me and contact links. So obviously I need to know which
data would you like to be created there.

**Comment by parasg1999** [comment]
I guess then these links should be removed for now, no point keeping them.

**Comment by ayushnanda21** [comment]
Sir, I have removed those two links and attached both views below. Please suggest if any modifications are required. Thanks.
Screenshot (587)
Screenshot (588)

**Pull request merged**